src/config/config-files-readme.txt

```diff
@@ -1,40 +0,0 @@
-Config files:
```

Comment:
I see that you've removed a lot of stuff from .../config/... as you said you would.  I thought that we wanted the readme to be kept.  I see the readme being removed, but I don't see it being "kept" somewhere else?